### PR TITLE
Adding text compression support to compress SDP messages.

### PR DESCRIPTION
**Pull request by akshay-prabhu-github**
**Issue #:**
Each time an attendee wishes to turn on/off their videos, switch resolutions or switch the number of videos being viewed, a SDP negotiation is required. This entails a full signaling Offer/Answer with SDP exchange. Each Unified Plan SDP media section contributes to ~4200 bytes of data. For an SDP with 25 media sections the size of the SDP comes up to 105KB. This increase in the network load can lead to bursty signaling-related network traffic which can impair audio and video quality. 

**Description of changes:**
Added compression support when sending and receiving sdp messages.

**Testing:**
1. Smoke tested locally enabling communication between my local backend.
2. Added Unit tests to validate the behavior.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
Yes. The demo application's functionality should not regress in terms of any functionality.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
no

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
Yes, it has been reviewed and approved.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -146,4 +147,4 @@
     "trailingComma": "es5",
     "printWidth": 100
   }
-}
+}
```
Comment (devalevenkatesh) [inline package.json:150]: nit: Needs a EOL new line. (Not sure how this got removed :) )

Comment (akshay-prabhu-github) [reply package.json]: strange I added that new EOL but it is not showing up in the review. 


check [ebfd1ce](https://github.com/aws/amazon-chime-sdk-js/pull/2053/commits/ebfd1ce650e05ce6185dba13609b2ed9e929ba61)

Comment (akshay-prabhu-github) [reply package.json]: I had to use *vim* editor to add the new line. Seems like my other editor was removing the new line.

File: package.json [hunk 2]
```diff
@@ -147,3 +148,4 @@
     "printWidth": 100
   }
 }
+
```
Comment (devalevenkatesh) [inline package.json:151]: So it should be there, not sure what is happening but basically this file needs only `pako` change 😅

Comment (akshay-prabhu-github) [reply package.json:151]: I am a little confused by your previous comment then. I checked the current file : https://github.com/aws/amazon-chime-sdk-js/blob/release-2.x/package.json

I did not see a new line there.

So should we add a new line or I should drop it?

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**